simulation/agents.py

```diff
@@ -496,9 +492,9 @@ def install_heating_system(
             if heating_system == HeatingSystem.HEAT_PUMP_GROUND_SOURCE:
                 self.boiler_upgrade_grant_used = 6_000
 
-    def update_heating_status(self, model: "DomesticHeatingABM") -> None:
+    def reset_previous_heating_decision_log(self) -> None:
```

Comment:
You could now call this in the `__init__` too if you wanted to? But happy as is.